### ISSUE TITLE
The browser does not load the favicon.ico

### ISSUE THREAD

**Issue by Gamodeur** [qdm12/ddns-updater#345]
Is this urgent: No
The browser does not load the favicon.ico

According to my research it comes from the "/" in the html code
![image](https://user-images.githubusercontent.com/15129966/180875457-729c186f-09f4-4051-b7aa-1d94e928a33f.png)

If this could be fixed it would be cool for my favorites bar 😊

**Comment by qdm12** [comment]
The Go code needs to serve an embedded favicon.ico on the /favicon.ico route, which is not the case for now.

**Comment by hugalafutro** [comment]
I returned to using ddns-updater after the integrated ddns thingy on router started crapping itself recently and noticed there ain't no favicon, but I could've sworn there was one when I used it before, the little blue gopher holding something iirc.
```
ddns-updater  | 2024/02/11 08:29:09 "GET http://192.168.1.101:8279/favicon.ico HTTP/1.1" from 192.168.1.163:5922 - 404 19B in 13.615µs
```

**Comment by qdm12** [comment]
Fixed with b70a91582c8e4f935924173d980addaa40fefb95
